app-text/qpdfview/qpdfview-0.4.18.recipe

```diff
@@ -77,12 +77,18 @@ INSTALL()
 	cd haiku_build
 	make install
 
+	mkdir -p $appsDir/qPDFView/data
```

Comment:
I don't really get why these three lines are needed. 'make install' should be told where to install at the qmake step.

Reply:
This is to create a folder corresponding to the folder "$appsDir/qPDFView/data" of "DATA_INSTALL_PATH" defined in "qpdfview.pri". Actually, the folder name ($appsDir -> ppsDir) is created on the working folder side with the first character deleted.

I created "qpdfview_ja.qm" and "help_ja.html" with the correct folder name to pass to the "Haiku" side.

Reply:
"Qpdfview.pri" seems to have two functions, not only the file placement function but also the function to inform the application of the location of the "translation file". Simply placing the file does not enable translation.

Reply:
I think it's because it supports various operating systems other than Linux. They have different file systems. Unfortunately, it doesn't seem to include a system like "Haiku" that is packaged and then installed.

Reply:
In the case of "haiku", I think "packaging" is not a "packaging" for installing a regular app, but a "cartridge" for mounting the app itself as a file system.

Reply:
Only the "mkdir" line was unnecessary.